examples/systemcalls/systemcalls.c

```diff
@@ -57,11 +62,42 @@ bool do_exec(int count, ...)
  *   (first argument to execv), and use the remaining arguments
  *   as second argument to the execv() command.
  *
-*/
+ */
+
+  if(command[0][0] != '/') 
```

Comment:
@qrogers6 it shouldn't be a requirement to use an absolute path, a relative path would be acceptable as well.